network-api/networkapi/wagtailpages/templates/wagtailpages/modular_page.html

```diff
@@ -34,11 +34,11 @@
     <div class="row my-5">
       <div class="{% block pre_footer_classes %}col-12 col-lg-10 offset-lg-1{% endblock %}">
         {% block signup %}{% endblock %}
-        <hr class="my-2">
-        <div class="opp-foot d-flex align-items-center"><img class="mr-3" src="/_images/green-20.svg" width="51" height="45">
-          <p class="mb-0">This is part of a broader movement for a healthy internet. <a href="/" id="see-more-modular-page">See more</a>.</p>
+        <div class="my-2 py-2 opp-foot">
```

Comment:
(assuming `opp-foot` stands for opportunity footer) Can we name this `ih-cta` or `internet-health-cta` as this shows up not just on opportunity pages but also campaign pages & this is wrapped in `pre_footer_classes` so it could sound weird if we name it `*-foot`.